keps/prod-readiness/sig-node/277.yaml

```diff
@@ -1,3 +1,5 @@
 kep-number: 277
+alpha:
+  approver: "@johnbelamaric"
 beta:
```

Comment:
can you remove this since we're not going to beta now?

Reply:
Yup, fixed!